tools/common/common-build.props

```diff
@@ -11,6 +11,7 @@
     <WindowsTargetPlatformMinVersion Condition="'$(WindowsTargetPlatformMinVersion)'==''">10.0.10586.0</WindowsTargetPlatformMinVersion>
     <EnableDotNetNativeCompatibleProfile Condition="'$(EnableDotNetNativeCompatibleProfile)'==''">true</EnableDotNetNativeCompatibleProfile>
     <IncludeOutputsInPackage>false</IncludeOutputsInPackage>
+    <IncludeFrameworkReferencesInPackage>false</IncludeFrameworkReferencesInPackage>
```

Comment:
><IncludeFrameworkReferencesInPackage>false</IncludeFrameworkReferencesInPackage> [](start = 4, length = 80)

Projects in tools.sln can importer common-build.props/targets, tools-build.props/targets, or neither. Does this needed to be added elsewhere as well?